nuopc.runconfig

```diff
@@ -40,31 +40,31 @@ PELAYOUT_attributes::
      esp_nthreads = 1
      esp_pestride = 1
      esp_rootpe = 0
-     glc_ntasks = 240
+     glc_ntasks = 1
      glc_nthreads = 1
      glc_pestride = 1
      glc_rootpe = 0
-     ice_ntasks = 240
+     ice_ntasks = 96 #Note-Parallel I/O github.com/COSIMA/access-om3/issues/214
```

Comment:
I think if https://github.com/payu-org/payu/pull/496 gets merged we don't need this note (which is technically also true for all the model components.)